dGame/dComponents/ControllablePhysicsComponent.cpp

```diff
@@ -84,14 +91,22 @@ void ControllablePhysicsComponent::Serialize(RakNet::BitStream* outBitStream, bo
 		m_DirtyCheats = false;
 	}
 
-	outBitStream->Write(m_DirtyPickupRadiusScale);
-	if (m_DirtyPickupRadiusScale) {
+	outBitStream->Write(m_DirtyEquippedItemInfo);
+	if (m_DirtyEquippedItemInfo) {
 		outBitStream->Write(m_PickupRadius);
-		outBitStream->Write0(); //No clue what this is so im leaving it false.
-		m_DirtyPickupRadiusScale = false;
+		outBitStream->Write(m_InJetpackMode);
```

Comment:
Is there any observed changes to jetpack with this change?

Reply:
well that value is never changed, soooooo no

Reply:
Will add functionality in future PR for this variable to be used by the jetpack behavior